### ISSUE TITLE
OG tags needed across the docs website

### ISSUE THREAD

**Issue by damedoteth** [ipfs/ipfs-docs#1483]
When sharing a link to a docs page on social media, there is no preview card displayed — ideally a card would appear that displays the page's title, description, image, etc. @2color

**Comment by ElPaisano** [comment]
@damedoteth I'm actually not sure if this is possible in vuepress

@johnnymatthews any thoughts on this?

**Comment by johnnymatthews** [comment]
Yeah this is doable. We just need to add something like this to the header of each page:

```html
<meta property="og:title" content="Installing IPFS">
<meta property="og:type" content="article" />
<meta property="og:description" content="A step-by-step guide for installing IPFS on your toaster.">
<meta property="og:image" content="https://docs.ipfs.tech/toaster/install.png">
<meta property="og:url" content="https://docs.ipfs.tech/toaster/install.html">
<meta name="twitter:card" content="install.png">
```

We could grab all this from the front-matter, expect the images which would likely come from some specific image used for all social cards.

While not overly complex, I'd wager the priority of something like this isn't particularly high. If someone from the community wants to pick this up though, this would be a nice addition to the docs.
